packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1129,6 +1129,30 @@ async function renderToHTMLOrFlightImpl(
       }
     }
 
+    // Per-segment prefetch data
+    //
+    // All of the segments for a page are generated simultaneously, including
+    // during revalidations. This is to ensure consistency, because it's
+    // possible for a mismatch between a layout and page segment can cause the
+    // client to error during rendering. We want to preserve the ability of the
+    // client to recover from such a mismatch by re-requesting all the segments
+    // to get a consistent view of the page.
+    //
+    // TODO (Per Segment Prefetching): This is placeholder data. Populate with
+    // the actual data generated during prerender.
+    const placeholder = Buffer.from(
+      'TODO (Per Segment Prefetching): Not yet implemented\n'
+    )
+    metadata.segmentFlightData = new Map([
+      // Root segment
+      ['/', placeholder],
+      ['/blog', placeholder],
```

Comment:
I can't tell from the diff but is this flagged?

Reply:
The whole feature is flagged by PPR but generating these output files isn't. So in the meantime it does generate extra stuff. I'll add the flag here, too.